From khali@linux-fr.org Sat Nov 26 19:24:45 2005
Date: Sat, 26 Nov 2005 20:51:06 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>, Russell King <rmk@arm.linux.org.uk>
Subject: [PATCH 17/25] i2c: Drop i2c_driver.{owner,name}, 8 of 11
Message-Id: <20051126205106.154b3d4d.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-08-acorn.patch

From: Laurent Riffard <laurent.riffard@free.fr>

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the drivers for acorn arch.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Acked-by: Russell King <rmk+kernel@arm.linux.org.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/acorn/char/pcf8583.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/acorn/char/pcf8583.c
+++ gregkh-2.6/drivers/acorn/char/pcf8583.c
@@ -257,7 +257,10 @@ pcf8583_command(struct i2c_client *clien
 }
 
 static struct i2c_driver pcf8583_driver = {
-	.name		= "PCF8583",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "PCF8583",
+	},
 	.id		= I2C_DRIVERID_PCF8583,
 	.attach_adapter	= pcf8583_probe,
 	.detach_client	= pcf8583_detach,
